### DISCUSSION TITLE
Prevent items from being destroyed by anything that uses the cactus damage type

### DISCUSSION THREAD

**Discussion by IcarussOne** [ACGaming/UniversalTweaks#203] [Tweak Suggestions]
What it says on the title! It would really make cactus farms less annoying to use

**Comment by Primitive-Human** [comment]
I love how Mojang thought that a cactus should behave as a black hole. 

And a cactus itself does not kill; the infections you get from the wounds are more likely to cause your death. 

I'd make a bit more sense to have a cactus cause mild weakness, poison, wither or nausea, because we don't have the "Infection" status effect.
